app/src/main/java/com/gathering/android/auth/signin/SignInViewModel.kt

```diff
@@ -37,15 +38,18 @@ class SignInViewModel @Inject constructor(
         signInRepository.signInUser(email, pass, onResponseReady = { state ->
             when (state) {
                 is ResponseState.Failure -> {
-                    // TODO show error
+                    _viewState.value =
+                        SignInViewState.Error.ShowGeneralError("can not reach the server")
```

Comment:
Error string can be provided by another component not being hard-coded in the code, however this is not required now, we will fix this later. Just for the record.